lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -861,6 +862,21 @@ protected function loadMappingFile($file)
         return $result;
     }
 
+    private function assertValid(string $file): void
```

Comment:
How about call it `validateMapping`?

Reply:
Could be less misleading since I don't use actual assertions here